msteams-platform/graph-api/rsc/grant-resource-specific-consent.md

```diff
@@ -440,6 +498,18 @@ For more information on how to get details of the apps installed in a specific t
 
 For more information on how to get details of apps installed in a specific chat, see [get the names and other details of apps installed in the specified chat](/graph/api/chat-list-installedapps#example-2-get-the-names-and-other-details-of-apps-installed-in-the-specified-chat).
 
+#### Check your app for added RSC permissions for a user
+
+1. Get the [user ID](../../bots/how-to/conversations/send-proactive-messages.md#get-the-user-id-team-id-or-channel-id).
```

Comment:
@v-bvishnu I don't think this is the right link to get a user's id.

One way to do it is through Graph using this API -- https://learn.microsoft.com/en-us/graph/api/user-get?view=graph-rest-1.0&tabs=http. A user's UPN can be passed to the request URL. The response body will then have an 'id' field that can be used as the user's id.

Alternatively, you don't need to pass the user's id at all in the GET /users/id/permissionGrants API. Instead of user ID, you can similarly pass the user's UPN, which our service will understand.

Reply:
Updated.